lobster/tools/codebeamer/codebeamer.py

```diff
@@ -172,7 +174,20 @@ def get_query(mh, cb_config, query_id):
 
     return rv
 
```

Comment:
Add one more blank line here. Classed and top-level functions should be separated by 2 blank lines.